### DISCUSSION TITLE
Required fields and properties for "ProfilePage" type

### DISCUSSION THREAD

**Discussion by glennyabellana** [schemaorg/schemaorg#3671] [Q&A]
Hello,

I have a question regarding schema markup for "ProfilePage" type. Are there required fields/properties based on this documentation https://schema.org/ProfilePage? There are properties that are red-colored fonts and there are blue-colored fonts. What are the differences for those properties?
<img width="802" alt="Screenshot 2024-10-07 at 8 14 10 AM" src="https://github.com/user-attachments/assets/70cd6f77-c9e9-4ca2-98fe-2385285ccb02">

Thanks!

**Comment by alvaropicho** [comment]
Hello @glennyabellana , I really don't know, but I investigate a little. With inspector tool, I see blue properties are in 'ext ext-pending' class, that means: "This term is in the "new" area - implementation feedback and adoption from applications and websites can help improve our definitions". So it's new.
Then, in red properties are 'core' class: definitely accepted... I attach an edited  screenshot.

Perhaps is about that; new vs. accepted
![blue-red](https://github.com/user-attachments/assets/114da59f-8e3c-427d-b553-0415eec66983)
